subgraphs/abracadabra/schema.graphql

```diff
@@ -37,7 +37,7 @@ enum ProtocolType {
 
 type Token @entity @regularPolling {
   " Smart contract address of the token "
-  id: ID!
+  id: Bytes!
```

Comment:
Just to confirm, the casing of IDs or IDs themselves do not change with this update to Bytes?

Reply:
@melotik ID values will change.
Eg, (interestRates) `"BORROWER-STABLE-0x13eca380e816edffc356b75b2fcd84b5d3c464d9-18900"` -> `0x00000000424f52524f5745522d535441424c452d13eca380e816edffc356b75b2fcd84b5d3c464d92d3138393030`

Reply:
My sense is that the downstream affects are too costly, so we should just leave it as is. But looking for confirmation from @jaksonkallio

Reply:
Reverted